src/components/Calculator.js

```diff
@@ -1,49 +1,103 @@
 import React from 'react';
 import Touch from './Touch';
 import Display from './Display';
+import calculate from '../util/logic/calculate';
 
 export default class Calculator extends React.Component {
```

Comment:
- [ ] Kindly add `÷` instead of one of the two pluses operations to match the design guidelines and requirements. Currently, you have two `+` signs in your calculator which doesn't make much sense.

![image](https://user-images.githubusercontent.com/24703171/206881536-5899689e-4b68-4f3c-9084-2c0239186dfe.png)

Reply:
Thanks very much